elements/pfe-clipboard-copy/pfe-clipboard-copy.scss

```diff
@@ -1,3 +1,272 @@
+:root {
```

Comment:
we shouldn't need to pull all this in. better to fall back

Reply:
I put that stuff in there just to play around with different workflows for doing 1:1 conversion.  After seeing some other examples I'm starting with the plain BaseClass for functionality and will add the PF styles later.